schema/cassandra/version.go

```diff
@@ -23,7 +23,7 @@ package cassandra
 // NOTE: whenever there is a new data base schema update, plz update the following versions
 
 // Version is the Cassandra database release version
-const Version = "0.28"
+const Version = "0.29"
```

Comment:
I think we can keep using the current version 0.28 as it's not released yet.

Reply:
Makes sense, I'll do it.